internal/tlsutil/tls.go

```diff
@@ -24,8 +24,27 @@ import (
 	"github.com/pkg/errors"
 )
 
-// NewClientTLS returns tls.Config appropriate for client auth.
-func NewClientTLS(certFile, keyFile, caFile string) (*tls.Config, error) {
+// Validate and return TLS renegotiation settings
+func GetRenegotiation(option string) (strategy tls.RenegotiationSupport, err error) {
+	switch option {
+	// RenegotiateNever disables renegotiation.
+	case "Never":
```

Comment:
a good idea might be to turn these cases into constant values. That way, we avoid a case where a typo may be used in the codebase. It also keeps the code more DRY.

`type ShowOutputFormat` is a good example to follow here. See https://github.com/helm/helm/blob/83847cc89980185ccbbd98747d7dc725e1e4910d/pkg/action/show.go#L30-L37

Reply:
Sorry if my code isn't that great, I'm very new to Golang and I'm still catching up on the best practices, patterns and style.